account_invoice_base_invoicing_mode/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "website": "https://github.com/OCA/account-invoicing",
     "license": "AGPL-3",
     "category": "Accounting & Finance",
-    "depends": ["account", "queue_job", "sale"],
```

Comment:
Why do you need sale_stock?

I think you are using at the test, but you could do it manually by edting the qty_delivered, isn't it?

Reply:
You shouldn't change dependencies of a stable module. If required, do a glue module.

Reply:
I is needed because of the refactoring, there was the dependency stock in account_invoice_mode_weekly/__manifest__.py declared.
But in fact it would also possible to remove this change and there for refactor also the unittests.